assets/src/components/editing/editor/normalizers/spaces.ts

```diff
@@ -2,10 +2,11 @@ import * as Immutable from 'immutable';
 import { Editor, Element, Path, Transforms } from 'slate';
 import { Model } from 'data/content/model/elements/factories';
 import { ModelElement } from 'data/content/model/elements/types';
+import { ModelTypes } from 'data/content/model/schema';
 import { FormattedText } from 'data/content/model/text';
 
-const spacesRequiredBetween = Immutable.Set<string>([
-  'image',
+export const spacesRequiredBetween = Immutable.Set<ModelTypes>([
+  'img',
```

Comment:
This rule was not firing between images due to a misspelled type here. Also adjusted the type definition so it can't happen again.